### PR TITLE
csr: Added support to load a license file

### PR DESCRIPTION
**Pull request by fredsod**

### CODE REVIEW

**Comment by plajjan** [comment]
The time & date we set for the VM is statically defined in the code, is that intentional? Are you setting a date in the past and it just works for a license that has been issued after this date? 😛

**Comment by fredsod** [comment]
... Yes :) The license does only have an expire date.

**Comment by plajjan** [comment]
For SROS I have to set a date that is after the issue date of the license file but ok, this is awesome. Can you just explain that in the README? I think it's worth pointing out!

File: csr/README.md [hunk 1]
```diff
@@ -75,6 +75,21 @@ RAM: 4GB
...
+License handling
+----------------
+You can feed a license file into CSR1000V by putting a text file containing the
+license in this directory next to your .qcow2 image. Name the license file the
+same as your .qcow2 file but append ".license", e.g. if you have
+"csr1000v-universalk9.16.04.01.qcow2" you would name the license file
+"csr1000v-universalk9.16.04.01.qcow2.license".
+
+The license is bound to a specific UDI and usually expires within a given time.
+To make sure that everything works out smoothly we are configure the clock to
```
Comment (plajjan) [inline csr/README.md]: "we are configure" << broken wording

**Comment by plajjan** [comment]
LGTM

**Pull request merged**